### DISCUSSION TITLE
Any information about `*.ns.uu.net` ?

### DISCUSSION THREAD

**Discussion by breezemight** [indianajson/can-i-take-over-dns#31] [Q&A] [answered]
`uu.net` is under Verizon. Can these ns servers be reached via Verizon small business or any other way?

**Comment by indianajson** [comment, marked as answer]
I believe they can be reached under Version's Business Hosting, but not Yahoo (now Verizon) Small Business. Sadly, the Business Hosting seems difficult to access as an individual.
